sdks/python/apache_beam/io/gcp/bigquery_tools.py

```diff
@@ -318,15 +319,20 @@ def get_query_location(self, project_id, query, use_legacy_sql):
 
     referenced_tables = response.statistics.query.referencedTables
     if referenced_tables:  # Guards against both non-empty and non-None
```

Comment:
Is implied by the `for` loop and can be dropped.

Reply:
@apilloud comment suggest that is also guarding against None value. If that's true, dropping this if, would break code `for loop` in case referenced_tables is actually None. 

Therefore my question is, are you sure we want to remove that if?

Reply:
You are right, this needs to stay. (I'm getting go and python mixed up here, I don't use either of them very often.)